### DISCUSSION TITLE
Bot and player in one time

### DISCUSSION THREAD

**Discussion by Debug-pro-dev** [PrismarineJS/mineflayer#1921] [Q&A] [answered]
Is it possible to connect a bot to an already active player?

That is, the player entered the server, logged in, and then the bot connects to it. The player can control his character, but for example, at the same moment, the bot responds to messages in the chat.

Yes, something like a hacked client.

(Before that, I created a discussion about the botfilter, but I thought that if there is a possibility from this discussion, then it is possible to bypass the botfilter in this way.)

**Comment by aesthetic0001** [comment]
I believe this is not possible with mineflayer

**Comment by Hazuki-san** [comment]
I doubt coding a server (flying-squid) + mineflayer API would work, but that takes time.

**Comment by amoraschi** [comment, marked as answer]
You can try modifying the proxy example in node-minecraft-protocol
